tests/run-centos-ci.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# setting up virtual environment
+yum -y install epel-release
+yum install libselinux-python yum-utils \
+  device-mapper-persistent-data \
+  lvm2 gcc python-virtualenv
+virtualenv --system-site-packages env
+source env/bin/activate
+
+# install dependency packages
+pip install ansible molecule docker-py
+
+# prerequisites to install docker
+yum-config-manager \
+    --add-repo \
+    https://download.docker.com/linux/centos/docker-ce.repo
+yum install docker-ce
+
+# start and enable Docker service
+systemctl start docker
+systemctl enable docker
+
+# basic dependencies for the tests
+yum -y install git
```

Comment:
You should remove these two lines. This will happen from the script inside centosci